release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
 
 You can now encrypt Azure storage accounts during installation by providing the installation program with a customer managed encryption key. See xref:../installing/installing_azure/installation-config-parameters-azure.adoc#installation-configuration-parameters-additional-azure_installation-config-parameters-azure[Installation configuration parameters] for descriptions of the parameters required to encrypt Azure storage accounts.
 
-[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
-==== CAPO integration into the cluster CAPI Operator (Tech Preview)
+[id="ocp-4-15-capi-openstack-tech-preview-no-upgrade"]
+==== OpenStack integration into the Cluster CAPI Operator (Tech Preview)
```

Comment:
The common-attributes.adoc file list:

:rh-openstack-first: OpenStack
:rh-openstack: OpenStack

Use these in place of hardcoded "OpenStack" references?

Reply:
I can use them for some but not all instances of "OpenStack", so I do that for the ones that make sense :+1: (I have a [card](https://issues.redhat.com/browse/OSDOCS-10442) to make attributes for the full "Cluster API Provider OpenStack" and all its friends, so I will leave those ones alone)